src/doc/src/reference/environment-variables.md

```diff
@@ -231,6 +231,7 @@ corresponding environment variable is set to the empty string, `""`.
 * `CARGO_PKG_RUST_VERSION` --- The Rust version from the manifest of your package.
   Note that this is the minimum Rust version supported by the package, not the
   current Rust version.
+* `CARGO_PKG_README` — Path to the README file of your package.
```

Comment:
See #11646

We use `---` to write em dash in markdown files. Tools will help us the conversion.

```suggestion
* `CARGO_PKG_README` --- Path to the README file of your package.
```

Reply:
@weihanglo I will fix that in a minute